debian/control

```diff
@@ -8,24 +8,23 @@ Build-Depends:  debhelper (>= 9),
                 libcurl4-openssl-dev,
                 libfontconfig1-dev,
                 libfreetype6-dev,
-                libssl-dev,
-                wget,
-                npm | nodejs
+                libssl-dev
 Standards-Version: 3.9.4
-Homepage: https://jellyfin.media/
+Homepage: https://jellyfin.org/
 Vcs-Git: https://github.org/jellyfin/jellyfin.git
 Vcs-Browser: https://github.org/jellyfin/jellyfin
 
-Package: jellyfin
+Package: jellyfin-server
 Replaces: mediabrowser, emby, emby-server-beta, jellyfin-dev, emby-server
 Breaks: mediabrowser, emby, emby-server-beta, jellyfin-dev, emby-server
 Conflicts: mediabrowser, emby, emby-server-beta, jellyfin-dev, emby-server
 Architecture: any
 Depends: at,
          libsqlite3-0,
-         jellyfin-ffmpeg,
+         jellyfin-ffmpeg (>= 4.2.1-2),
```

Comment:
Should we bump the version? We're on -7 now

Reply:
Not needed - it's just a greater or equal, since that -2 added the last incompatible change. We can still support -3 for example. Since only -7 is in the repo any longer, that's the only version it could get anyways.